docker-compose.yml

```diff
@@ -1,7 +1,37 @@
-version: "2"
+
 services:
+    ngircd:
+        image: lscr.io/linuxserver/ngircd:latest
+        container_name: ngircd
+        environment:
+            - PUID=1000
+            - PGID=1000
+            - TZ=Etc/UTC
+        volumes:
+            - ./.docker/ngircd/config:/config
```

Comment:
do we really need this? I ran it just fine with the default config

Reply:
I actually do not know. I tested without it first, got into issues, added the config, and so on. But you might be correct. I'll check it again.